js/main.js

```diff
@@ -274,30 +274,15 @@ function addTables() {
     dst_at_start = moment([current.year(), current.month(), current.date(), options.work_day_start]).isDST();
   }
 
-  var row, section;
-  var size_class = "";
-  if (options.stops.length > 1) {
-    size_class = "col-lg-12";
-  }
   // There are two separate calls here,
   $.ajax({
     url: options.url + "stops/get/" + options.stops[stop_index],
     success: function(stop_info) {
-      if (stop_index % 2 == 0) {
-        row = $('<div class="row route-holder"></div>');
-      } else {
-        row = $('.route-holder:last');
-      }
-      section = $('<div class="col-xs-24 ' + size_class + '"></div>"');
-      row.append(section);
-      if (stop_index % 2 == 0) {
-        container.append(row);
-      }
```

Comment:
Many of our installations rely on the two-column layout to fit more stops on a wide screen (for instance, note the photo that Jonathan sent).

Reply:
Is that configured through a local config file?

Reply:
Never mind, I see it now. It's weird that these rows are out of 24 rather than 12.